Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -61,6 +62,15 @@ GameFile::GameFile(const QString& path) : m_path(path)
   }
 
   m_valid = true;
+
+  static Core::TitleDatabase title_database;
```

Comment:
Not convinced we need this to be static.
Making this a static also prevents any language change from taking effect until you restart Dolphin.